.gitmodules

```diff
@@ -1,9 +1,3 @@
 [submodule "envoy-filters/envoy"]
 	path = envoy-filters/envoy
 	url = https://github.com/envoyproxy/envoy.git
```

Comment:
Can't we remove all the submodules?

Reply:
fixed with: https://github.com/wso2/product-microgateway/pull/3245/commits/593f3343d74d0e8b9c8c1cc60637c28d3e5076d5

Reply:
If we are removing this, let's remove files in the path `<product-microgateway>/envoy-filters/envoy` as well.

Reply:
removed with : https://github.com/wso2/product-microgateway/pull/3245/commits/aa94416781242d52347b747a4942483308279e61